### DISCUSSION TITLE
Overseer running at an absolute crawl

### DISCUSSION THREAD

**Discussion by aad4321** [sct/overseerr#3665] [General]
Hello, curious how the performance is with people running the latest version. I have not really changed anything on my environment and running this via Docker. When using the app it is painfully slow requiring multiple refreshes and at points its completely unresponsive. Anything I can do to help improve this?

**Comment by sbuxkid152** [comment]
@aad4321 just an FYI that I'm new to Overseerr and experienced the same thing when I first installed it on docker (I'm using a Synology). I'm also running all the starr apps on the same system and was getting database wait errors, etc. To solve this, I added an SSD volume in my synology and moved my appdata files for all docker apps onto the SSD. The difference I experienced was like 10000000%. Everything on all apps became zippy and just worked very well. I was skeptical when reading forum posts about adding SSD storage just for these apps and didn't think it would make that much of a difference, but boy does it. I suggest trying that; worked for me.

**Comment by nmbrg** [comment]
Not sure if I’m blind or if they’re just ignoring this issue.
Haven’t found any mentions of it beeing worked on and overseerr is unusable at the moment. And all these so called fixes/workaround are useless since most of them I’m already using (like only ssd in my nas etc) or tried without success.

I’d hope they fix the bug but seems that won’t happen anytime soon. I really do not want to go back to ombi, but soon I have no other option. :(

**Reply by aad4321** [reply]
I redeployed to a cloud vps running docker. It's significantly faster.

**Comment by nmbrg** [comment]
> I redeployed to a cloud vps running docker. It's significantly faster.

ok. im not paying a vps to have overseerr working as intended. :) then ill just move to another solution instead.
and besides that, im not sure what a vps would do for me since im on 250mbit connection and a synology rs1619xs+ has more than enough of "juice" to handle overseerr.

and to add to that, ive never had any issues with overseerr before upgrading to the latest (or was it the version before that? hm..). and havent changed anything else with my infrastructure or anmything that could make this happen. so im pretty confident that the issue is with overseer and not my hardware/connection/software/OS etc..

anyhow, im not here to argue, just to add that i also has issues with overseerr and hopefully well get a working version real soon.
if not, theres other solutions out there.

**Reply by aad4321** [reply]
i recall trying to redeploy the container with the same results. the only thing i did not do is completely wipe the data and set it up net new on the NAS. I only did this on the VPS and the performance restored. Maybe try deploying a new container with new storage and see how it performs? I feel like an upgrade is what slowed it down but a fresh install may fix it.

**Reply by nmbrg** [reply]
i did set up a new fresh test of overseerr container and nothing changed. still unusable.

**Comment by randadinata** [comment]
It's constantly reading something

![image](https://github.com/sct/overseerr/assets/6295471/bf63a695-a195-4697-8174-d23f439b54ce)

**Comment by ericblade** [comment]
I have to restart docker every few days to keep Overseerr responsive.  Just restarting the container does not help.  

None of the other containers I have have the same problem, and seem to run fine for weeks at a time. 

There's nothing obviously wrong in the logs (just usual log chatter that happens) and it doesn't seem to be consuming a massive amount of CPU (holding at 0% according to top inside the container, despite that it is seriously dragging very slowly right now), it just starts dragging and dragging and dragging to the point where almost nothing loads.
